### DISCUSSION TITLE
What is the release schedule

### DISCUSSION THREAD

**Discussion by bcorner13** [vesta-webtrees-2-custom-modules/vesta_common#98] [General]
I updated my webtrees today, including php version to 8, and found that the vesta suite was not compatible.
`This version of the module is incompatible with your current webtrees version. Only webtrees versions before 2.1.14 are supported. All functionality of the module is therefore disabled. To restore functionality, either use an updated version of this module (once available) or downgrade your webtrees version (if possible).
 Custom module
 2.1.13.0.0
 Richard Cissée
 https://cissee.de/`

**Comment by ric2016** [comment]
[Webtrees unfortunately doesn't use proper semantic versioning](https://github.com/vesta-webtrees-2-custom-modules/vesta_common/issues/97), so I have to check compatibility for every release.
This usually happens shortly after each webtrees release. I plan to do this today for 2.1.14.

Note that you get a warning in the webtrees upgrade wizard regarding module incompatibility - In general, the safest option is to wait a few days when upgrading to the latest webtrees version, unless there is an urgent security-related issue.

**Comment by ric2016** [comment]
Vesta release 2.1.15.0.0 is available now (note that there was yet another webtrees release in the meantime ...)
